std/os/windows.zig

```diff
@@ -756,11 +756,19 @@ pub fn sliceToPrefixedSuffixedFileW(s: []const u8, comptime suffix: []const u16)
     return result;
 }
 
+inline fn MAKELANGID(p: USHORT, s: USHORT) LANGID {
+    return (s << 10) | p;
+}
+
 /// Call this when you made a windows DLL call or something that does SetLastError
 /// and you get an unexpected error.
 pub fn unexpectedError(err: DWORD) std.os.UnexpectedError {
     if (std.os.unexpected_error_tracing) {
-        std.debug.warn("unexpected GetLastError(): {}\n", err);
+        var buf: LPSTR = undefined;
```

Comment:
I think you're passing an undefined pointer to the function. It might help to use the underlying type in the variable declaration rather than the Windows type alias name to make it obvious. I think you want `&buf` in the function call and the variable to be an array.